nautobot/nautobot_data_validation_engine/filters.py

```diff
@@ -0,0 +1,164 @@
+            "content_type__model": "iexact",
+            "field": "iexact",
+            "regular_expression": "icontains",
+        }
+    )
+    content_type = ContentTypeMultipleChoiceFilter(
+        choices=FeatureQuery("custom_validators").get_choices,
+        conjoined=False,  # Make this an OR with multi-values
+    )
+
+    class Meta:
+        """Filterset metadata for the RegularExpressionValidationRule model."""
+
+        model = RegularExpressionValidationRule
+        fields = "__all__"
+
+
+class MinMaxValidationRuleFilterSet(NautobotFilterSet):
+    """Base filterset for the MinMaxValidationRule model."""
+
+    q = SearchFilter(
+        filter_predicates={
+            "name": "icontains",
+            "error_message": "icontains",
+            "content_type__app_label": "iexact",
+            "content_type__model": "iexact",
+            "field": "iexact",
+        }
+    )
+    content_type = ContentTypeMultipleChoiceFilter(
+        choices=FeatureQuery("custom_validators").get_choices,
+        conjoined=False,  # Make this an OR with multi-values
+    )
+
+    class Meta:
+        """Filterset metadata for the MinMaxValidationRuleFilterSet model."""
+
+        model = MinMaxValidationRule
+        fields = "__all__"
+
+
+class RequiredValidationRuleFilterSet(NautobotFilterSet):
+    """Base filterset for the RequiredValidationRule model."""
+
+    q = SearchFilter(
+        filter_predicates={
+            "name": "icontains",
+            "error_message": "icontains",
+            "content_type__app_label": "iexact",
+            "content_type__model": "iexact",
+            "field": "iexact",
+        }
+    )
+    content_type = ContentTypeMultipleChoiceFilter(
+        choices=FeatureQuery("custom_validators").get_choices,
+        conjoined=False,  # Make this an OR with multi-values
+    )
+
+    class Meta:
+        """Filterset metadata for the RequiredValidationRuleFilterSet model."""
+
+        model = RequiredValidationRule
+        fields = "__all__"
+
+
+class UniqueValidationRuleFilterSet(NautobotFilterSet):
+    """Base filterset for the UniqueValidationRule model."""
+
+    q = SearchFilter(
+        filter_predicates={
+            "name": "icontains",
+            "error_message": "icontains",
+            "content_type__app_label": "icontains",
+            "content_type__model": "icontains",
+            "field": "iexact",
+        }
+    )
+    content_type = ContentTypeMultipleChoiceFilter(
+        choices=FeatureQuery("custom_validators").get_choices,
+        conjoined=False,  # Make this an OR with multi-values
+    )
+
+    class Meta:
+        """Filterset metadata for the UniqueValidationRuleFilterSet model."""
+
+        model = UniqueValidationRule
+        fields = "__all__"
+
+
+#
+# DataCompliance
+#
+
+
+class CustomContentTypeFilter(filters.MultipleChoiceFilter):
+    """Filter for ContentType that doesn't rely on the model's plural name to be in the registry."""
```

Comment:
What's this about? Something not right with the existing content-type filter variants?

Reply:
Feels like the filtering logic is very similar to `ContentTypeMultipleChoiceFilter` in `core.filters`